background.js

```diff
@@ -0,0 +1,17 @@
+function contextMenu() {
+	chrome.contextMenus.create({
+		"title": "Generate QR from Page URL",
```

Comment:
no need for `"` when your object key has only allowed symbols (e.g. it doesnt contain spaces)